charts/k8s-monitoring/templates/agent_config/_profiles_ebpf.river.txt

```diff
@@ -57,8 +57,10 @@ discovery.relabel "ebpf_pods" {
 
 pyroscope.ebpf "ebpf_pods" {
   targets = discovery.relabel.ebpf_pods.output
-  forward_to = [pyroscope.write.profiles_service.receiver]
+
   demangle = {{ .Values.profiles.ebpf.demangle | quote }}
+
+  forward_to = [pyroscope.write.profiles_service.receiver]
```

Comment:
Nit: Not an improvement, most likely committed by accident.

Reply:
This was more just my own style. When defining the fields inside a component, I like to do:
{
// inputs (where does the data come from)

// processing (what do you do with the data)

// delivery (where does the data go)
}

Reply:
I see and there it all makes sense 🙂